### PR TITLE
Facilitator 1920 application initial setup

### PR DESCRIPTION
**Pull request by clareconstantine**
(to be merged first)

- Create `FacilitatorApplicationBase` and `Facilitator1920Application` models, moving common code into the base class.
- Create new pd factories and `Facilitator1920Application` unit tests
- Create shared facilitator application constants file and `Facilitator1920Application` constants file, moving shared constants into the shared file (for now, left all constants the same until we go in to update the facilitator app questions)

### CODE REVIEW

File: dashboard/test/models/pd/application/facilitator1920_application_test.rb [hunk 1]
```diff
@@ -0,0 +1,332 @@
+require 'test_helper'
+require 'state_abbr'
```
Comment (mehalshah) [inline dashboard/test/models/pd/application/facilitator1920_application_test.rb]: What is this require statement for?

File: dashboard/test/models/pd/application/facilitator1920_application_test.rb [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+      assert_queries 0 do
+        assert_equal @fit_workshop, @application_with_fit_workshop.fit_workshop
+
+        # also prefetches assigned workshop
+        assert_equal @workshop, @application_with_fit_workshop.workshop
+      end
+    end
+
+    test 'fit_cohort' do
+      skip "update when 1920 fit registration is implemented"
```
Comment (mehalshah) [inline dashboard/test/models/pd/application/facilitator1920_application_test.rb:290]: Suggest removing this test instead of using the skip

**Review by mehalshah — APPROVED** [review]

**Comment by clareconstantine** [comment]
[Begin setting up 2019-20 facilitator application template](https://trello.com/c/So57SbEB/764-begin-setting-up-2019-20-facilitator-application-template)

File: dashboard/test/models/pd/application/facilitator1920_application_test.rb (hunk 2 above)
Comment (clareconstantine) [reply dashboard/test/models/pd/application/facilitator1920_application_test.rb:290]: I want to keep it so I don't forget and we lose a test - and I am implementing 1920 fit reg tomorrow, so this shouldn't end up checked in and spamming

File: dashboard/test/models/pd/application/facilitator1920_application_test.rb (hunk 1 above)
Comment (clareconstantine) [reply dashboard/test/models/pd/application/facilitator1920_application_test.rb]: good catch, definitely not using it.

**Pull request merged**